frontend/src/lib/components/SaveToDashboard/SaveToDashboard.tsx

```diff
@@ -29,8 +29,6 @@ export function SaveToDashboard(props: Props): JSX.Element {
     let _filters: Record<string, any> | null = null
     let _annotations: Array<Record<string, any>> | null = null
 
-    const _type: string = item.type
```

Comment:
We still need to pass this type to display or handle it on the backend because from line 98 on `frontend/src/insights` we calculate what the display should be using `determineInsightType` which is passed into type. 

(Adding retention table or graph as dashboard item will break right now because on the insights page it uses "ActionsTable" to figure out which chart to display to keep the chartFilter simple but when you save it to dashboard it needs to be saved using "RetentionTable" so that the dashboard item uses the right container component)

Reply:
I've removed determineInsightType completely. We were already setting display anyway so didn't need this